### PR TITLE
[new downloader][android] fix: New downloader fixes

### PR DESCRIPTION
**Pull request by trashkalmar**

### CODE REVIEW

**Comment by trashkalmar** [comment]
@ygorshenin @mpimenov PTAL

File: android/src/com/mapswithme/maps/DownloadResourcesActivity.java [hunk 1]
```diff
@@ -152,10 +152,10 @@ public void onLocationUpdated(Location l)
   private final Listener mResourcesDownloadListener = new Listener()
   {
     @Override
-    public void onProgress(final int progress)
+    public void onProgress(final int percents)
```
Comment (mpimenov) [inline android/src/com/mapswithme/maps/DownloadResourcesActivity.java]: s/percents/percentage/
или
s/percents/percent/

**Comment by mpimenov** [comment]
LGTM after fix

**Comment by ygorshenin** [comment]
LGTM

**Comment by yunikkk** [comment]
LGTM

**Pull request merged**